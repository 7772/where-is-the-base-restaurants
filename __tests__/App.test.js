import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

describe('App Test', () => {
  let App;

  beforeEach(() => {
    App = require('../src/App').default;
  });

  it('renders correctly', () => {
    const json = renderer.create(<App />).toJSON();

    expect(json).toMatchSnapshot();
  });
});
