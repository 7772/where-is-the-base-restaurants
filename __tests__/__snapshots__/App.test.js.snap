// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test renders correctly 1`] = `<View />`;
